packages/thirdweb/package.json

```diff
@@ -195,6 +195,7 @@
     "@radix-ui/react-icons": "1.3.0",
     "@radix-ui/react-tooltip": "1.1.2",
     "@tanstack/react-query": "5.29.2",
+    "@thirdweb-dev/merkletree": "^0.2.6",
```

Comment:
actually instead of a dep, can you just copy the entire source of this package inside the thirdweb package, and replace our current MerkeTree.js file with it?

Reply:
hmm by doing so we will have to import these deps:
```
  "dependencies": {
    "buffer": "^6.0.3",
    "buffer-reverse": "^1.0.1",
    "treeify": "^1.1.0"
  },
```

Reply:
no, instead we have to fix the logic in `merkletree.js` (without adding these dependencies)

Reply:
`merkletree.js` is already based on the same file, so somewhere there is a bug, we have to hunt it down and fix it

Reply:
@kien-ngo remove